Dependencies/napi/napi-jsi/include/napi/napi.h

```diff
@@ -39,6 +39,8 @@ typedef enum {
   napi_uint32_array,
   napi_float32_array,
   napi_float64_array,
+  napi_bigint64_array,
+  napi_biguint64_array,
```

Comment:
These weren't included since JSI doesn't support bigint. Maybe I should have commented them out with a comment explaining.